src/pug/blog/shop_opening.pug

```diff
@@ -0,0 +1,54 @@
+
+  p Additionally it is not environmentally friendly (nor cheap) for everyone to place minimum orders for components, for example the minimum amount of PCBs to order in JLCPCB is 5 (2 of them soldered and 3 of them raw), and the minimum amount of Protopasta conductive filament to buy is 50 grams while the Alpakka reference design only needs 4 grams, so quite a waste in general. On the top of that, some of the components are almost impossible to source in some regions, for example getting M2.5 #[em millimeters] bolts in the United States.
+
+  p Offering DIY kits and parts in the smaller quantities will reduce waste.
+
+  p And finally there is the funding reason, one of #[+a('/about') Input Labs principles] is #[em to create a stable environment for research and development], as you may guess, this is not easy when doing open source and Creative Commons, one of our support streams is #[+ax(org.patreon) Patreon], which is doing great, but we need some additional help, as hopefully the "shop" will do, to create such stable environment.
+  hr
+
+  h3 So we are selling controllers now?
+  p Not really, for now we are only offering some selected parts, the more problematic for people to source or build. And even if we offer a bigger variety of DIY kits and parts later, we may never sell assembled controllers.
+  p Our project is different because you can mix and match reference parts and custom parts for building your dream controller, where would be the fun and accessibility if you could just buy a boring standard controller of the shelf?
+  p At some point (assuming people do not need customizations) it may be possible to buy parts and assemble the reference designs Ikea-style, but meatballs won't be included, best we can do is some finnish Salmon soup, which is clearly the superior nordic food ;)
+  hr
+
+  h3 What's the impact for people sourcing their materials from somewhere else?
+  p Absolutely none, people will be still able to grab the designs from the web or GitHub, and build the controllers with components and materials of their choice. The DIY kits available on the Input Labs Distribution portal are just another option on the menu, not meant to replace the freedom of building and modifying your controller in any way you want.
+  hr
+
+  h3 What will be the quality of the parts you offer?
+  p For some components we will be just re-selling parts that we buy in bigger batches, for example the M2.5 bolts or electronic components. We won't post-process these in any way, so the quality is the same you could expected from the original manufacturers.
+
+  p Other parts we will actually 3D-print them, these won't be at the same level of precision you would expect from an industrially manufactured / injection molded product. Since these are for DIY purposes, the final user may need to do some tweaks to fit perfectly in their personal project.
```

Comment:
"We will 3D print parts ourselves, so you should not expect the quality of an industrially manufactured product, such as the usual injection molded components."